DatadogLogs/Sources/Logger.swift

```diff
@@ -7,247 +7,183 @@
-    func remove(tag: String)
-}
-
-public extension Logger {
-    /// Sends a DEBUG log message.
-    /// - Parameters:
-    ///   - message: the message to be logged
-    ///   - error: the error information (optional)
-    ///   - attributes: a dictionary of attributes (optional) to add for this message. If an attribute with
-    /// the same key already exist in this logger, it will be overridden (only for this message).
-    func debug(_ message: String, error: Error? = nil, attributes: [AttributeKey: AttributeValue]? = nil) {
-        log(level: .debug, message: message, error: error, attributes: attributes)
-    }
-
-    /// Sends an INFO log message.
-    /// - Parameters:
-    ///   - message: the message to be logged
-    ///   - error: the error information (optional)
-    ///   - attributes: a dictionary of attributes (optional) to add for this message. If an attribute with
-    /// the same key already exist in this logger, it will be overridden (only for this message).
-    func info(_ message: String, error: Error? = nil, attributes: [AttributeKey: AttributeValue]? = nil) {
-        log(level: .info, message: message, error: error, attributes: attributes)
-    }
-
-    /// Sends a NOTICE log message.
-    /// - Parameters:
-    ///   - message: the message to be logged
-    ///   - error: the error information (optional)
-    ///   - attributes: a dictionary of attributes (optional) to add for this message. If an attribute with
-    /// the same key already exist in this logger, it will be overridden (only for this message).
-    func notice(_ message: String, error: Error? = nil, attributes: [AttributeKey: AttributeValue]? = nil) {
-        log(level: .notice, message: message, error: error, attributes: attributes)
-    }
+/// Datadog logger.
+public struct Logger {
+    public struct Configuration {
+        /// Format to use when printing logs to console.
+        public enum ConsoleLogFormat {
+            /// Prints short representation of log.
+            case short
+            /// Prints short representation of log with given prefix.
+            case shortWith(prefix: String)
+        }
 
-    /// Sends a WARN log message.
-    /// - Parameters:
-    ///   - message: the message to be logged
-    ///   - error: the error information (optional)
-    ///   - attributes: a dictionary of attributes (optional) to add for this message. If an attribute with
-    /// the same key already exist in this logger, it will be overridden (only for this message).
-    func warn(_ message: String, error: Error? = nil, attributes: [AttributeKey: AttributeValue]? = nil) {
-        log(level: .warn, message: message, error: error, attributes: attributes)
+        /// The service name  (default value is set to application bundle identifier)
+        public var serviceName: String?
+
+        /// The logger custom name (default value is set to main bundle identifier)
+        public var loggerName: String?
```

Comment:
suggestion/ Following [rule 7](https://datadoghq.atlassian.net/wiki/spaces/RUMP/pages/3027638506/Public+API+Convention+in+iOS+SDK+2.x#Rule-7%3A-Use-of-standard-Datadog-and-RUM-terminology) and _“avoid platform self referencing”_ I rather think it should be `name`. It is used as part of `Logger.create()` so it's rather obvious what the `name` would be. WDYT?
```suggestion
        /// The logger custom name (default value is set to main bundle identifier)
        public var name: String?
```

Reply:
Agreed!